core/win32/os_private.h

```diff
@@ -331,7 +331,7 @@ os_rename_file_in_directory(IN HANDLE rootdir,
 /* thread-shared only needs 4 pages on 32-bit but -thread_private needs 5
  * in case we hook the image entry on an early cbret
  */
-#define INTERCEPTION_CODE_SIZE IF_X64_ELSE(8*4096,7*4096)
+#define INTERCEPTION_CODE_SIZE IF_X64_ELSE(9*4096,7*4096)
```

Comment:
Let's add a comment explaining that we need extra space for dr_syscall_intercept_natively.